app/main.py

```diff
@@ -202,17 +201,18 @@ def rabbit_status_callback(self, response):
             status = res.get('status')
             logger.info("Rabbit MQ health check response {}".format(status))
             if status == "ok":
-                logger.info('Setting status now')
+                logger.info("Checked rabbit health", health="ok")
                 self.rabbit_status = True
 
     def determine_ftp_status(self):
         try:
             self.ftp_status = False
             conn = self.ftp.get_connection()
             if conn:
+                logger.info("Checked FTP health", health="ok")
```

Comment:
Can health ever be anything else but ok?

Reply:
No... but I think the health is either ok or an exception is being thrown.  @necrophonic do you have any opinions as these changes were suggested by you?  My original change changed this line from something silly to 'FTP health ok' so it reflected what was happening

Reply:
All the messages should be the same really - the only difference should be the actual health value and optionally an exception message that covers the context

Reply:
so on an exception/error there should be a health="bad" field?

Reply:
or maybe health="not ok", I'm still not sure what sounds better

Reply:
Really depends what you're healthchecking. In PaaS land, "unhealthy" means the app will be restarted. If the FTP is liable to come back (and a restart of the app isn't going to help) then technically the app is "healthy" even though it's not doing things and will return errors to clients calling it.